Client/game_sa/CProjectileInfoSA.cpp

```diff
@@ -46,6 +46,11 @@ void CProjectileInfoSA::RemoveProjectile(CProjectileInfo* pProjectileInfo, CProj
         // Has it not already been removed by GTA?
         if (pProjectileInfo->IsActive())
         {
+            // Fix crash when disconnecting
+            // GH #3876
+            if (!projectileInfoInterface->pEntProjectileOwner || projectileInfoInterface->pEntProjectileOwner->nType == ENTITY_TYPE_NOTHING || projectileInfoInterface->pEntProjectileOwner->pReferences == reinterpret_cast<void*>(0x80808080))
+                bBlow = false;
```

Comment:
You need to find a reason why pEntProjectileOwner was freed.
It looks like MTA doesn't delete all projectiles while disconnecting.

Reply:
This is not the goal of this PR. This PR is only a temporary solution to prevent the game from crashing. If you want, feel free to investigate the cause and open a PR with the fix, and then I'll close mine. For now, this PR is just a temporary crash patch.

Reply:
Temporary solutions have a high chance to become permanent.
The solution is too dirty. We don't rush with this issue.